kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/network/model/OrderSpendResult.java

```diff
@@ -0,0 +1,186 @@
+package com.kin.ecosystem.network.model;/*
+ * Kin Ecosystem
+ * Apis for client to server interaction
+ *
+ * OpenAPI spec version: 1.0.0
+ *
+ *
+ * NOTE: This class is auto generated by the swagger code generator program.
+ * https://github.com/swagger-api/swagger-codegen.git
+ * Do not edit the class manually.
+ */
+
+import com.google.gson.TypeAdapter;
+import com.google.gson.annotations.JsonAdapter;
+import com.google.gson.annotations.SerializedName;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonWriter;
+import java.io.IOException;
+
+/**
+ * OrderSpendResult
+ */
+@JsonAdapter(OrderSpendResult.Adapter.class)
+public class OrderSpendResult {
+
+    /**
+     * Gets or Sets type
+     */
+    @JsonAdapter(TypeEnum.Adapter.class)
+    public enum TypeEnum {
```

Comment:
there an easter way to serialise/deserialise enum http://blog.jensdriller.com/simple-deserialization-of-java-enums-using-google-gson-annotations/

Reply:
it's generated from swagger